conductr_cli/conduct_load.py

```diff
@@ -155,6 +157,26 @@ def validate(cache_dir, default_cache_dir):
     validate(configuration_cache_dir, DEFAULT_CONFIGURATION_RESOLVE_CACHE_DIR)
 
 
+def is_bundle(input):
+    return os.path.isfile(input) and zipfile.is_zipfile(input)
+
```

Comment:
Simple determination for now. This could be improved as validation is implemented.